fastlane/lib/fastlane/supported_platforms.rb

```diff
@@ -1,17 +1,27 @@
 module Fastlane
   class SupportedPlatforms
+    class << self
+      attr_accessor :extra
+      attr_reader :default
+
+      def extra=(value)
+        value ||= []
+        UI.important("Setting '#{value}' as extra SupportedPlatforms")
+        @extra = value
+      end
+    end
+
+    @default = [:ios, :mac, :android]
+    @extra = []
+
     def self.all
-      [
-        :ios,
-        :mac,
-        :android
-      ]
+      (@default + @extra).flatten
     end
 
-    # this will throw an exception if the passed platform is not supported
+    # this will log a warning if the passed platform is not supported
     def self.verify!(platform)
```

Comment:
one could consider removing the trailing slash here. Or keeping it for Backwards Compatibility only and adding a new method without it.